src/Client/Web/Bit.Client.Web.BlazorUI/Components/ChoiceGroup/BitChoiceGroupOption.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bit.Client.Web.BlazorUI.Components.ChoiceGroup
+{
+    public class BitChoiceGroupOption
```

Comment:
Plz add `AriaLabel`, `LabelId` and `Id` prop like [fluent](https://developer.microsoft.com/en-us/fluentui#/controls/web/choicegroup)